src/actors/ball.ts

```diff
@@ -0,0 +1,61 @@
+import * as ex from 'excalibur';
+import {GameEvent} from "excalibur";
```

Comment:
In terms of style generally encouraged to have space in single line curly braces `{}` like `{ GameEvent }`